tools/go.mod

```diff
@@ -17,6 +18,10 @@ tool (
 	mvdan.cc/gofumpt
 )
 
+// use local submodule path to prevent the error
+// `module declares its path as: benchmarking but was required as: github.com/FerretDB/mongodb-benchmarking`
```

Comment:
Or we could just fix it: https://github.com/idealo/mongodb-benchmarking/pull/35

Reply:
Thank you :pray: